### DISCUSSION TITLE
Assign shortcut to zoom in/zoom out the terminal?

### DISCUSSION THREAD

**Discussion by volkan0x** [zed-industries/zed#10599] [Feature Requests]
How to assign shortcut to zoom in/zoom out the terminal?

<img width="836" alt="Screenshot 2024-04-16 at 11 33 51" src="https://github.com/zed-industries/zed/assets/68243739/8533ad64-a538-4add-95a7-308541e68c25">

**Comment by SomeoneToIgnore** [comment]
Thank you for spotting, I've added the keybinding labels on those shortcuts in https://github.com/zed-industries/zed/pull/10604
![image](https://github.com/zed-industries/zed/assets/2690773/5099555d-8992-4005-ab59-5b1f580954b3)

The ToggleZoom action is universal across all panes and is mentioned in the default keybindings:
```
"shift-escape": "workspace::ToggleZoom",
```
you can override it in the local bindings, if needed.

**Reply by fcFn** [reply]
Took me a bit to understand what that icon means. I didn't know that it meant Esc. Thanks!

**Reply by dmahlow** [reply]
I came here looking for this answer, as I hadn't seen this icon for ESC before either.

**Reply by sumitbhavra** [reply]
oh boy! this was ESC

**Reply by enderahmetyurt** [reply]
It's ESC 🫠 it took my 15 mins to understand in this morning 😅 Thank you
